src/melee/lb/lb_00F9.c

```diff
@@ -446,28 +457,28 @@ HSD_CObj* lb_80013B14(HSD_CameraDescPerspective* desc)
     return cobj;
 }
 
-bool lb_80013BB0(void)
+bool lb_80013BB0(ColorOverlay* arg)
 {
     return true;
 }
 
-int lb_80013BB8(struct lb_80013FF0_t* arg0)
+bool lb_80013BB8(ColorOverlay* arg0)
 {
-    arg0->x0 += *arg0->unk_8 & 0x03FFFFFF;
-    ++arg0->unk_8;
-    return 0;
+    arg0->x0_timer += *((s32*) (void*) &arg0->x8_ptr1->light_color) & 0x03FFFFFF;
```

Comment:
I suspect there's a better way to do this, but when I tried accessing individual members of `light_color`, it kept compiling to `lbz` instead of `lbw` 🤔

Reply:
Don't worry about it, I've matched it locally.

Reply:
https://github.com/ribbanya/melee/blob/496215d9746da6267a96e35d5607b55560c5f1bb/src/melee/lb/lb_00F9.c#L715-L720

Reply:
This is better, though:
https://github.com/ribbanya/melee/blob/d61c62724940a0ec53516e7860168819edee916d/src/melee/lb/lb_00F9.c#L715-L720

Reply:
And the union:
https://github.com/ribbanya/melee/blob/d61c62724940a0ec53516e7860168819edee916d/src/melee/lb/types.h#L376-L379